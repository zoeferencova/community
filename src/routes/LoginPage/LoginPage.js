import React, { Component } from "react";

export default class LoginPage extends Component {
    render() {
        return (   
            <h1>LoginPage</h1>
        )
    }
}