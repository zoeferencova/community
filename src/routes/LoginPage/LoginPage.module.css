.form {
    width: 80%;
}

.form h1 {
    font-size: 22px;
    font-weight: 500;
    margin-bottom: 30px;
}

.form p,
.form p a {
    color: #8e97a3;
    font-size: 15px;
}