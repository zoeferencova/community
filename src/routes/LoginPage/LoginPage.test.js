import React from "react";
import { render } from "enzyme";
import LoginPage from "./LoginPage";
import { BrowserRouter } from "react-router-dom";
import CommUnityContext from "../../contexts/context";

import mockContext from "../../contexts/mock-context";

describe("LoginPage", () => {
    it("renders LoginPage component given mock context data", () => {
        const wrapper = render(
            <CommUnityContext.Provider value={mockContext}>
                <BrowserRouter>
                    <LoginPage />
                </BrowserRouter>
            </CommUnityContext.Provider>
        );
        
        expect(wrapper).toMatchSnapshot();
    });
});