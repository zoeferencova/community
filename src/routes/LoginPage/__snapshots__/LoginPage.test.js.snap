// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage renders LoginPage component given mock context data 1`] = `
<div
  class="container"
>
  <div
    class="main"
  >
    <h1>
      Sign in
    </h1>
    <div
      class="form"
    >
      <form
        class="form"
      >
        <label
          class="label "
          for="email"
        >
          Email
        </label>
        <input
          class="input "
          id="email"
          name="email"
          required=""
          type="text"
          value=""
        />
        <label
          class="label "
          for="password"
        >
          Password
        </label>
        <input
          class="input "
          id="password"
          name="password"
          required=""
          type="password"
          value=""
        />
        <div
          class="buttonContainer"
        >
          <button
            class="button "
            type="button"
          >
            Cancel
          </button>
          <button
            class="buttonDark "
            loading="false"
            type="submit"
          >
            Sign in
          </button>
        </div>
      </form>
      <p>
        Don't have an account? 
        <a
          href="/register"
        >
          Sign up
        </a>
      </p>
    </div>
  </div>
</div>
`;
